docs/reference/modules/discovery/zen.asciidoc

```diff
@@ -82,6 +82,12 @@ serves as a protection against (partial) network failures where node may unjustl
 think that the master has failed. In this case the node will simply hear from
 other nodes about the currently active master.
 
+If the setting `discovery.zen.master_election.filter_client` is `true`, pings from client nodes (or non-data,
```

Comment:
Instead of non-data non-master-eligible, i'd just use the settings:  "from nodes which node `node.client` set to `true`, or both `node.data` and `node.master` set to `false`.

And similarly for data nodes.